### ISSUE TITLE
Docfx not working on apple silicon

### ISSUE THREAD

**Issue by bartofzo** [dotnet/docfx#9081]
Running docfc gives the message:

There is no template processing document type(s): Conceptual,ManagedReference,Toc

If I run the command docfx template list, it gives me nothing.
Also when I run docfx template export default, it says:
default is not an embedded template.

My build succeeds but the website is not served. The _site folder only contains images, manifest.json and xrefmap.yml.

**Expected behavior**
The site should build

**Context (please complete the following information):**
- OS: MacOS 13.2.1, Apple M2 Pro
- Docfx version: 2.70.0

**Additional context**
The problems began when switching to a macbook pro M2. First I had this problem but patched it:
https://github.com/dotnet/docfx/issues/9047

Before on an intel mac everthing worked fine with the same project.

**Comment by bartofzo** [comment]
Some additional info.

When I type:
docfx template list,

I get the following error:
_Error: Could not find a part of the path '/opt/homebrew/Cellar/docfx/2.70.0/libexec/templates'._

I've reinstalled docfx and now when building I get message for every API source file saying:
_xxxxxx.yml: No template bundle found for ManagedReference, model will be ignored._

Same goes for resources: 
_No template bundle found for Resource, model will be ignored_

I've also tried just pasting the templates directory from the docfx source in my project and adjusting docfx.json to use a template from there. I'm using default + statictoc.

I then get these warnings:
_xxxx/docfx_project/ManagedReference.html.primary.js: warning: Not a valid template preprocessor, ignored: Cannot read property 'path' of null
xxxx/RestApi.html.primary.js: warning: Not a valid template preprocessor, ignored: Cannot read property 'path' of null
xxxx/UniversalReference.html.primary.js: warning: Not a valid template preprocessor, ignored: Cannot read property 'path' of null
xxxx/conceptual.html.primary.js: warning: Not a valid template preprocessor, ignored: Cannot read property 'path' of null_

And the output of the site is jibberish. All of the API documents are empty. The articles contain the text I wrote but no other layout.

Also, I'm having the same issue as:
https://github.com/dotnet/docfx/issues/9047
So it's worth mentioning that I first need to change the $PATH and copy libhostfxr.dylib as described here:
https://github.com/microsoft/vscode-dotnettools/issues/120#issuecomment-1640892593

All of these things may be related. Hoping somebody can be of help here! I need to update documentation for an existing project and I'm completely stuck.

Also wondering if there is anybody on Apple Silicon for who docfx works as expected?

**Comment by filzrev** [comment]
> Error: Could not find a part of the path '/opt/homebrew/Cellar/docfx/2.70.0/libexec/templates'.

It seems you are using docfx that installed with `homebrew`.

**docfx formula**
From https://formulae.brew.sh/formula/docfx

**installation scripts(docfx.rb)**
https://github.com/Homebrew/homebrew-core/blob/bf6baa7f9fcf26f5e132133d430840fcbbfa0ee6/Formula/d/docfx.rb

This `homebrew formula` seems some step are missing to build templates.
So it fails to load template when running docfx command.

**Workaround**
> I've also tried just pasting the templates directory from the docfx source.

You needs to copy `templates` that are already build. Check [Release Page's zip file](https://github.com/dotnet/docfx/releases/tag/v2.70.0`).

or simply use `.NET global tools version` or `Self-Contained version` that published on GitHub release page
to resolve problems.

**Comment by bartofzo** [comment]
After copying the templates from the build into my project, it works. Thank you!

**Comment by kaiaschulz** [comment]
Okay, as I had the same issue and some problems to follow your recommendations, here is the solution I used.
I installed [docfx with homebrew](https://formulae.brew.sh/formula/docfx).

When I tried to run (`docfx ./web/docfx.json;docfx serve ./web/docfxwebexport`) my project, I got the following warning:
`warning: No template bundles were found, no template will be applied to the documents. 1) Check your docfx.json 2) the templates subfolder exists inside your application folder or your docfx.json directory.`

I downloaded the newest osc .zip-file ([docfx-osx-x64-v2.78.3.zip](https://github.com/dotnet/docfx/releases/download/v2.78.3/docfx-osx-x64-v2.78.3.zip)) of the [docfx release page](https://github.com/dotnet/docfx/releases).
Afterwards, I extracted the .zip-file and copied the `template`-folder to `/opt/homebrew/Cellar/docfx/2.78.3/libexec`.
Now, I could run docfx without issues :)

Thanks a lot.
